File: [iOS] HEALTH 1ST/maps.ios-master/NMapSampleObjc/NMapSampleObjc/MainViewController.m
//
//  MainViewController.m
//  NMapSampleObjc
//
//  Created by Naver on 2016. 11. 11..
//  Copyright © 2016년 Naver. All rights reserved.
//

#import "MainViewController.h"
#import "PolylinesViewController.h"
#import "PolygonsViewController.h"
#import "CirclesViewController.h"
#import "CustomMarkersMapViewController.h"
#import "LocationMapViewController.h"
#import "ReverseGeocoderViewController.h"
#import "MovableMarkerViewController.h"
#import "RotationViewController.h"

@interface MainViewController ()

@end

@implementation MainViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
    switch (indexPath.row) {
        case 1:
            {
                LocationMapViewController * viewController = [[LocationMapViewController alloc] init];
                viewController.title = @"내위치";
                [self.navigationController pushViewController:viewController animated:YES];
            }
            break;
        case 2:
            {
                ReverseGeocoderViewController * viewController = [[ReverseGeocoderViewController alloc] init];
                viewController.title = @"주소찾기";
                [self.navigationController pushViewController:viewController animated:YES];
            }
            break;
        case 4:
             {
             MovableMarkerViewController * viewController = [[MovableMarkerViewController alloc] init];
             viewController.title = @"마커 그리기";
             [self.navigationController pushViewController:viewController animated:YES];
             }
            break;
       case 5:
            {
                PolylinesViewController * viewController = [[PolylinesViewController alloc] init];
                viewController.title = @"선 그리기";
                [self.navigationController pushViewController:viewController animated:YES];
            }
            break;
        case 6:
            {
                PolygonsViewController * viewController = [[PolygonsViewController alloc] init];
                viewController.title = @"면 그리기";
                [self.navigationController pushViewController:viewController animated:YES];
            }
            break;
        case 7:
            {
                CirclesViewController * viewController = [[CirclesViewController alloc] init];
                viewController.title = @"원 그리기";
                [self.navigationController pushViewController:viewController animated:YES];
            }
            break;
        default:
            break;
    }
}

@end

File: [iOS] HEALTH 1ST/maps.ios-master/NMapSampleSwift/NMapSampleSwift/MainViewController.swift
//
//  ViewController.swift
//  NMapSampleSwift
//
//  Created by Naver on 2016. 11. 8..
//  Copyright © 2016년 Naver. All rights reserved.
//

import UIKit

class MainViewController: UITableViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
        switch(indexPath.row) {
        case 1:
            let viewController = LocationMapViewController()
            viewController.title = "내위치"
            self.navigationController?.pushViewController(viewController, animated: true)
        case 2:
            let viewController = ReverseGeocoderViewController()
            viewController.title = "주소찾기"
            self.navigationController?.pushViewController(viewController, animated: true)
        case 4:
            let viewController = MovableMarkerViewController()
            viewController.title = "이동 가능한 마커 그리기"
            self.navigationController?.pushViewController(viewController, animated: true)
        case 5:
            let viewController = PolylinesViewController()
            viewController.title = "폴리라인 그리기"
            self.navigationController?.pushViewController(viewController, animated: true)
        case 6:
            let viewController = PolygonsViewController()
            viewController.title = "폴리곤 그리기"
            self.navigationController?.pushViewController(viewController, animated: true)
        case 7:
            let viewController = CirclesViewController()
            viewController.title = "원 그리기"
            self.navigationController?.pushViewController(viewController, animated: true)
        default:
            break
        }
    }

}
